### ISSUE TITLE
Cannot load resource

### ISSUE THREAD

**Issue by yanns** [ktoso/sbt-jmh#104]
First of all, thanks for this sbt plugin!

I have a little problem:
If I try to load a resource like:
```
getClass.getClassLoader.getResourceAsStream("my-file.json")
```

with `my-file.json` inside `resources`,

-> this works with a `Main extends App`
-> but does not work for a JMH benchmark.

I guess that the resources are not available for the `jmh:run` sbt scope.

**Comment by rockwotj** [comment]
Did you figure out a workaround? I tried adding `resourceDirectories in Jmh += (resourceDirectory in Compile).value,` but that didn't seem to help

**Comment by rockwotj** [comment]
It seemed: `resourceDirectory in Jmh := (resourceDirectory in Compile).value` fixed this however. Not sure why the former didn't work!

**Comment by agourlay** [comment]
@rockwotj thank you for the workaround 👍

**Comment by yanns** [comment]
thx @rockwotj!
This is a configuration we could put into the plugin

**Comment by ktoso** [comment]
Happy to merge a PR adressing these issues if any of you have the time to
make one :)

-- 
Konrad 'ktoso’ Malawski
Akka <http://akka.io> @ Lightbend <http://lightbend.com>
<http://java.pl/>java.pl / geecon.org / krakowscala.pl / lambdakrk.pl /
sckrk.com

On 23 February 2017 at 18:30:17, Yann Simon (notifications@github.com)
wrote:

thx @rockwotj <https://github.com/rockwotj>!
This is a configuration we could put into the plugin

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/ktoso/sbt-jmh/issues/104#issuecomment-281940989>, or mute
the thread

**Comment by yanns** [comment]
https://github.com/ktoso/sbt-jmh/pull/109

**Comment by ktoso** [comment]
Releasing fix right now, update to 0.2.23
